runtime/karura/src/lib.rs

```diff
@@ -1628,6 +1654,7 @@ impl Convert<MultiLocation, Option<CurrencyId>> for CurrencyIdConvert {
 			X3(Parent, Parachain(id), GeneralKey(key)) => {
 				match (id, &key[..]) {
 					(parachains::bifrost::ID, parachains::bifrost::BNC_KEY) => Some(Token(BNC)),
+					(parachains::bifrost::ID, parachains::bifrost::VSKSM_KEY) => Some(Token(VSKSM)),
```

Comment:
will be good to add `BNC` and `VSKSM` conversions into integration tests https://github.com/AcalaNetwork/Acala/blob/ea45a0301539e2d4d6947178a0c910343ade1d64/runtime/integration-tests/src/integration_tests.rs#L1428

Reply:
BNC and VSKSM conversions are already added into integration tests. Please reveiw.